utils/measure/measure.py

```diff
@@ -911,23 +911,45 @@ def create(self) -> PowerMeter:
 def main():
     print(f"Powercalc measure: {_VERSION}\n")
 
+    device = inquirer.list_input("What kind of device do you want to measure the power of?",
```

Comment:
Let's introduce an enum (`StrEnum`) for device type.

Reply:
Not sure what they do nor how to do it? Did not get any wiser by googling either.. so need help

Reply:
It's considered bad practice to repeat the same string twice and check for string values in if statements.
These should be constants or enums.
Enums are relatively new in Python but exists in many other languages. 
The idea is to create an enumaration (list) of possible values.

You can create an enum by subclassing `StrEnum`. See [here](https://github.com/bramstroker/homeassistant-powercalc/blob/master/custom_components/powercalc/const.py#L157) for some examples where I use it as well.
So you can add an `DeviceType` enum.
Also you can include all enum values automatically in the inquirer list `[cls.value for cls in DeviceType]`

So that means when a device type is added we only have to add a new value to the enum.

In the if statements you can check `device == DeviceType.LIGHT` for example. When an invalid enum value is passed an error will be raise. Also your IDE (pycharm, visual studio) will do autocompletion.

So you see a lot of advantages for clear and consise code.